src/pages/profile/index.js

```diff
@@ -0,0 +1,251 @@
+import { Route, useNavigate } from "react-router-dom";
+import Card from "../../components/card";
+import TextInput from "../../components/form/textInput";
+import "./style.css";
+import "../../styles/index.css";
+import LockIcon from "../../assets/icons/locIcon";
+import EyeIcon from "../../assets/icons/eyeIcon";
+import useAuth from "../../hooks/useAuth";
+import jwt_decode from "jwt-decode"
+import { useEffect, useState } from 'react'
+import { get } from '../../service/apiClient'
+
+
+const hrStyle = {
+  backgroundColor: "#e6ebf5",
+  height: 1,
+};
+
+const ViewProfile = () => {
+  const navigate = useNavigate();
+  const [ userId, setUserId ] = useState()
+	const { token } = useAuth();
+
+  const handleClick = () => {
+    navigate("/profile/1/edit");
+  };
+
+  const getUserInfo = async (userId) => {
+    get()
+  }
+  // getUserInfo()
```

Comment:
This looks unused. `get()` is going to cause an error, let's remove this for now

Reply:
Removed as requested.